server/client_manager.py

```diff
@@ -567,6 +575,40 @@ def wtce_mute(self):
             self.wtce_counter = (self.wtce_counter + 1) % times_per_interval
             self.wtce_time[self.wtce_counter] = time.time()
             return 0
+        
+        def ooc_mute(self):
+            """
+            Check if the client can use OOC or not.
+            :returns: how many seconds the client must wait to use OOC
+            """
+            if self.is_mod or self in self.area.owners:
+                return 0
+            if self.ooc_mute_time:
+                if (
+                    time.time() - self.ooc_mute_time
+                    < self.server.config["ooc_floodguard"]["mute_length"]
```

Comment:
This will prevent the server from launching if server owners haven't updated their configuration yet.

Reply:
It will not prevent the server from launching, a default of 0 is set in client_manager.py in line 106